### ISSUE TITLE
C1_W2_Lab_1_beyond_hello_world.ipynb: some portions still tailored for mnist dataset

### ISSUE THREAD

**Issue by rediet-orange** [https-deeplearning-ai/tensorflow-1-public#68]
In the exploration exercises some portions are still pertaining to mnist, instead of fashion_mnist.
- `mnist = tf.keras.datasets.mnist` -> `mnist = tf.keras.datasets.fashion_mnist`
  - Exercises 2, 4, 5, and 7: no impact on the global meaning, just for consistency
  - Exercise 3: the dimensions in the error message don't match the explanation
  - Exercise 6: just might have an issue with the 128-neuron hidden layer not having the same behavior as the baseline 512-neuron layer in terms of loss evolution
- Exercise 8, the 95% in the text should be changed to 60% (95% was for mnist)

**Comment by cfav-dev** [comment]
Hi! Thank you for the feedback! The end-of-lab exercises have now been revised to use fashion_mnist.
